drivers/planar/clarity_matrix.cr

```diff
@@ -0,0 +1,111 @@
+module Planar; end
+
+require "placeos-driver/interface/powerable"
+
+# Documentation: https://aca.im/driver_docs/Planar/020-1028-00%20RS232%20for%20Matrix.pdf
+#  also https://aca.im/driver_docs/Planar/020-0567-05_WallNet_guide.pdf
+
+class Planar::ClarityMatrix < PlaceOS::Driver
+  include Interface::Powerable
+
+  # Discovery Information
+  generic_name :VideoWall
+  descriptive_name "Planar Clarity Matrix Video Wall"
+
+  #   implements :device # don't need?
+
+  def on_load
+    # Communication settings
+    queue.wait = false
+    transport.tokenizer = Tokenizer.new("\r")
+  end
+
+  def on_unload
+  end
+
+  def on_update
+  end
+
+  def connected
+    schedule.every(60.seconds) { do_poll }
+    do_poll
```

Comment:
`schedule.every` takes an optional second arg. It this is `true` it will run the scheduled block straight away in addition to the schedule so that you don't need to repeat like this.